core/src/test/resources/h2/schema-h2.sql

```diff
@@ -143,10 +143,7 @@ CREATE TABLE IF NOT EXISTS `role_meta` (
     `role_name` VARCHAR(128) NOT NULL COMMENT 'role name',
     `metalake_id` BIGINT(20) UNSIGNED NOT NULL COMMENT 'metalake id',
     `properties` MEDIUMTEXT DEFAULT NULL COMMENT 'schema properties',
-    `securable_object_full_name` VARCHAR(256) NOT NULL COMMENT 'securable object full name',
-    `securable_object_type` VARCHAR(32) NOT NULL COMMENT 'securable object type',
-    `privileges` VARCHAR(64) NOT NULL COMMENT 'securable privileges',
-    `privilege_conditions` VARCHAR(64) NOT NULL COMMENT 'securable privilege conditions',
+    `securable_objects` VARCHAR(2048) NOT NULL COMMENT 'securable objects',
```

Comment:
I'm afraid 2048 may not be enough for lots of securable_objects if the role is a big role.

Reply:
Change to TEXT.